day_two_guide/topics/proc_backing-up-project.adoc

```diff
@@ -13,18 +13,20 @@ Module included in the following assemblies:
 Creating a backup of all relevant data involves exporting all important
 information, then restoring into a new project.
 
-[NOTE]
+[IMPORTANT]
 ====
-Currently, a {product-title} project back up and restore tool is being developed
-by Red Hat. See the following bug for more information:
-
-* https://bugzilla.redhat.com/show_bug.cgi?id=1303205[bugzilla 1303205].
+The following procedure utilizes the `oc get all` command, which does not list
+all resources. The command `oc api-resources --namespaced=true -o name` will
```

Comment:
This really should be steps later in the procedure, like:

. List all potential objects in the namespace:

. Check which objects are in the namespace:

. Back up the objects:

If not, please set the note to present tense and replace "may" with "might" or "can," as required. And rewrite the last sentence to be active and not use "below."

Reply:
Okay, I wasn't sure if this would be better as a "Hey this needs your attention" or steps. I'll add this to the procedure.